Datastore/src/Operation.php

```diff
@@ -710,11 +793,18 @@ public function mutation(
      */
     public function rollback($transactionId)
     {
-        $this->connection->rollback([
+        list($data, $optionalArgs) = $this->splitOptionalArgs([
             'projectId' => $this->projectId,
             'transaction' => $transactionId,
             'databaseId' => $this->databaseId,
```

Comment:
We don't need to call `splitOptionalArgs` here, since we already know none of the items in the array we are defining are valid for `$optionalArgs`. Just define this array as `$data` and pass in `[]` instead of passing `$optionalArgs` below.